### ISSUE TITLE
Typos in docs that need correcting:

### ISSUE THREAD

**Issue by JMercerGit** [TI-Tokyo/riak-docs-fork#283]
1. on `/riak/kv/3.2.0/configuring/next-gen-replication/` under `ttaaefs_maxresults` there appears to be part of a word cut off at the start of the description:

"or tictac full-sync what is the maximum number of AAE segments to be compared per exchange. Reducing this will speed up clock compare queries, but will increase the number of exchanges required to complete a repair."

**Comment by pjaclark** [comment]
Moved to https://www.tiot.jp/riak-docs-beta/riak/kv/3.2.0/configuring/next-gen-replication/reference/#fullsync-on-source-cluster

Should be `For` not `or`:

```
For Tictac full-sync this is the maximum number of AAE segments to be compared per exchange. Reducing this will speed up clock compare queries, but will increase the number of exchanges required to complete a repair.
```

**Comment by pjaclark** [comment]
On that same page, all references to `tictac` should refer to `TicTac`.

**Comment by JMercerGit** [comment]
Thanks peter

**Comment by JMercerGit** [comment]
These issues are solved in PR #284
